docs/tutorials/01_ui/viz_ui_slider.py

```diff
@@ -37,13 +37,43 @@ def cube_maker(color=(1, 1, 1), size=(0.2, 0.2, 0.2), center=(0, 0, 0)):
 cube = cube_maker(color=(0, 0, 1), size=(20, 20, 20), center=(15, 0, 0))
 
 ###############################################################################
-# Now we'll add two sliders: one circular and one linear.
+# Now we'll add two sliders: 1 circular and 4 linear sliders.
 
 ring_slider = ui.RingSlider2D(center=(630, 400), initial_value=0,
                               text_template="{angle:5.1f}°")
 
-line_slider = ui.LineSlider2D(center=(400, 230), initial_value=0,
-                              min_value=-10, max_value=10)
+hor_line_slider_text_top = ui.LineSlider2D(center=(400, 230), initial_value=0,
+                                           orientation='horizontal',
+                                           min_value=-10, max_value=10,
+                                           alignment='top')
+
+hor_line_slider_text_bottom = ui.LineSlider2D(center=(400, 200),
+                                              initial_value=0,
+                                              orientation='horizontal',
+                                              min_value=-10, max_value=10,
+                                              alignment='bottom')
+
+ver_line_slider_text_left = ui.LineSlider2D(center=(100, 400), initial_value=0,
+                                            orientation='vertical',
+                                            min_value=-10, max_value=10,
+                                            alignment='left')
+
+ver_line_slider_text_right = ui.LineSlider2D(center=(150, 400),
+                                             initial_value=0,
+                                             orientation='vertical',
+                                             min_value=-10, max_value=10,
+                                             alignment='right')
+
+###############################################################################
+# By default the alignment are 'bottom' for horizontal and 'top' for vertical
```

Comment:
You can move this text to the section above. after the following text: `4 linear sliders.`